test/vast_parser.js

```diff
@@ -1210,6 +1210,68 @@ describe('VASTParser', function() {
     });
   });
 
+  describe('#Nester xml extensions', () => {
+    let ad;
+    before(done => {
+      vastParser
+        .getAndParseVAST(urlfor('nested-xml-extensions.xml'), {
+          urlhandler: nodeURLHandler
+        })
+        .then(response => {
+          ad = response.ads[0];
+          done();
+        });
+    });
+
+    it('should have 2 extensions', () => {
+      ad.extensions.should.have.length(2);
+    });
+
+    it('validate first extension', () => {
```

Comment:
maybe it would also be good to check that the extension doesn't have a name or value attribute